src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int RANDOM_CEIL = 101;
+    private final ColorSupplier supplier = new ColorSupplier();
+    private final Random random = new Random();
+
+    public Ball getRandomBall() {
+        return new Ball(supplier.getRandomColor(), random.nextInt(RANDOM_CEIL));
```

Comment:
It's better to create local variables for color and number and then provide to constructor

Reply:
Thank you for your advice, I appreciate this!